src/Illuminate/Support/Str.php

```diff
@@ -1415,6 +1415,10 @@ public static function upper($value)
      */
     public static function title($value)
     {
+        if (is_null($value)) {
```

Comment:
You can avoid using a function call:

```suggestion
        if ($value === null) {
```